python_modules/dagster/dagster_tests/definitions_tests/declarative_automation_tests/scenario_utils/base_scenario.py

```diff
@@ -592,13 +592,18 @@ def run(
 
 
 def run_request(
-    asset_keys: Sequence[CoercibleToAssetKey],
+    asset_keys: Union[AssetKey, Sequence[CoercibleToAssetKey]],
```

Comment:
several callsites were passing a single asset key here, resulting in iterating over the `AssetKey` tuple which works fine but i would guess was not intended.

I think iterating over the asset key tuple now being disallowed is the riskiest part of the current PR which I think is fine since its very unlikely that it was what was intended